domains/client/domain-executor/src/core_bundle_processor.rs

```diff
@@ -250,4 +256,25 @@ where
         self.domain_block_processor
             .deduplicate_and_shuffle_extrinsics(parent_hash, extrinsics, shuffling_seed)
     }
+
+    fn filter_invalid_xdm_extrinsics(&self, exts: Vec<Block::Extrinsic>) -> Vec<Block::Extrinsic> {
```

Comment:
I'm wondering how will we prove that an invalid xdm extrinsic is included in a bundle to the farmers?

Reply:
The execution trace will be different and fraud proof will be generated for that specific extrinsinc index. What are you thinking ?

Reply:
Make sense to me as discussed offline.